static/docs/commands-reference/diff.md

```diff
@@ -54,10 +86,50 @@ Git reference.
 
 ```dvc
     $ dvc diff HEAD^
-    dvc diff from 6a819f8fa053f124f6a5487efc824a8c17366c71 to 612f6caf5c5daeb172167db285efd8b169d41b60
+    dvc diff from df613bce6dc0738f71c62d1748e1edfb3b7e4893 to ed10968bcb0dab72563d05712f24ddfff698c87b
```

Comment:
let's create a ticket (feel free to do it if you feel like it, confirm with @efiop first) to print shorter versions of git commits ... check how git is doing this, especially is it doing something different if a shorter version conflicts

Reply:
Done: https://github.com/iterative/dvc/issues/1902